### PR TITLE
feat(clusters): configure operators pull secrets for standalone clusters

### PR DESCRIPTION
**Pull request by machi1990**
Follows up https://github.com/bf2fc6cc711aee1a0c2a/kas-installer/pull/161#discussion_r859157081

## Description

Make sure that standalone clusters operators secret is properly referenced when set. This follows up  https://github.com/bf2fc6cc711aee1a0c2a/kas-installer/pull/161#discussion_r859157081
/cc @MikeEdgar can you have a look?

## Verification Steps

Verification from @MikeEdgar 

## Checklist (Definition of Done)

- ~~[ ] All acceptance criteria specified in JIRA have been completed~~
- ~~[ ] Unit and integration tests added that prove the fix is effective or the feature works (tested against emulated and non-emulated OCM environment)~~
- ~~[ ] Documentation added for the feature~~
- [ ] CI and all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer
- ~~[ ] Required metrics/dashboards/alerts have been added (or PR created).~~
- ~~[ ] Required Standard Operating Procedure (SOP) is added.~~
- ~~[ ] JIRA has created for changes required on the client side~~

### CODE REVIEW

File: internal/kafka/internal/workers/clusters_mgr.go [hunk 1]
```diff
@@ -1011,7 +1010,7 @@ func (c *ClusterManager) buildImagePullSecret(namespace string) *k8sCoreV1.Secre
 			Kind:       "Secret",
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      imagePullSecretName,
+			Name:      kafkaConstants.ImagePullSecretName,
 			Namespace: namespace,
 		},
 		Type: k8sCoreV1.SecretTypeDockercfg,
```
Comment (MikeEdgar) [inline internal/kafka/internal/workers/clusters_mgr.go:1016]: Not part of this PR of course, but should this be `SecretTypeDockerConfigJson` instead? Just based on the secret key name of `image-pull.dockerconfigjson`.

**Review by MikeEdgar — APPROVED** [review]
This looks good to me, thank you @machi1990

File: internal/kafka/internal/workers/clusters_mgr.go (hunk 1 above)
Comment (machi1990) [reply internal/kafka/internal/workers/clusters_mgr.go:1016]: Probably yes, and `SecretTypeDockerConfigJson` is the new format. So we should be using it instead.  

I am not sure we'd want to change it in this PR though.

Comment (MikeEdgar) [reply internal/kafka/internal/workers/clusters_mgr.go:1016]: > I am not sure we'd want to change it in this PR though.

Agreed - thanks for confirming though
